apps/vscode/vscode.py

```diff
@@ -48,13 +48,13 @@ def filename():
 
 @ctx.action_class("edit")
 class edit_actions:
-    def find(text: str):
+    def find(text = None):
```

Comment:
```suggestion
    def find(text: Optional[str] = None):
```

Reply:
I don't think the type signature is necessary here, as the type signature is already given when the action is declared. However, looking at actions.list() it looks like the type signature is incorrect:

```
edit.find(text: str = None)
  Open Find dialog, optionally searching for text
```

Reply:
Filed an issue: https://github.com/talonvoice/talon/issues/295

Reply:
Fair enough 🙂